license-information.md

```diff
@@ -1,29 +1,41 @@
+- A license, for example, [dotnet/runtime LICENSE.TXT](https://github.com/dotnet/runtime/blob/main/LICENSE.TXT).
+- Third party notice file, for example, [dotnet/runtime THIRD-PARTY-NOTICES.TXT](https://github.com/dotnet/runtime/blob/main/THIRD-PARTY-NOTICES.TXT)
+
+Third party notice files are used to document source copied (and possibly further modified) from other projects.
 
 [Project copyright guidance](https://github.com/dotnet/runtime/blob/main/docs/project/copyright.md) provides more details on our policies.
 
 ## Binaries
 
-Project binaries (like the .NET runtime distribution) primarily use the MIT license. Binaries built for macOS and Linux exclusively use the MIT license.
+Microsoft built binaries use the MIT license, for:
+
+- NuGet packages
+- Runtime (including ASP.NET Core) and SDK builds for Linux and macOS
+
+
+[Windows builds](license-information-windows.md) carry additional license terms, for closed source dependencies. These terms apply to runtimes (including ASP.NET Core), NuGet runtime packs, and SDK builds, but not to NuGet library packages.
+
+See [.NET Asset Licensing Model](https://github.com/dotnet/runtime/blob/main/docs/project/licensing-assets.md) for more information.
 
-[Windows builds](license-information-windows.md) carry additional license terms, for closed source dependencies.
+Note: The licensing of .NET builds provided by other parties may differ.
 
 ## Redistribution
 
 Binaries produced by .NET SDK compilers (C#, F#, VB) can be redistributed without additional restrictions. The only restrictions are based on the license of the compiler inputs used to produce the binary.
 
-Parts of the .NET runtime are embedded in applications, including [platform-specific executable hosts](https://learn.microsoft.com/dotnet/core/deploying/deploy-with-cli#framework-dependent-executable), and [self-contained deployments](https://learn.microsoft.com/dotnet/core/deploying/deploy-with-cli#self-contained-deployment), are subject to [.NET](https://github.com/dotnet/dotnet/blob/main/LICENSE.TXT) and [third-party notice](https://github.com/dotnet/dotnet/blob/main/THIRD-PARTY-NOTICES.txt) license terms.
+Parts of the .NET runtime are embedded in applications, including [platform-specific executable hosts](https://learn.microsoft.com/dotnet/core/deploying/deploy-with-cli#framework-dependent-executable),
+and [self-contained deployments](https://learn.microsoft.com/dotnet/core/deploying/deploy-with-cli#self-contained-deployment), are subject to [.NET](https://github.com/dotnet/dotnet/blob/main/LICENSE.TXT) and [third-party notice](https://github.com/dotnet/dotnet/blob/main/THIRD-PARTY-NOTICES.txt) license terms.
 
 Binaries that target Windows are subject to [additional terms](license-information-windows.md).

```

Comment:
This applies to runtime only. It does not apply to NuGet packages with Windows-specific libraries, like `Microsoft.Win32.Registry`.